packages/apps/job-launcher/server/src/modules/cron-job/cron-job.service.ts

```diff
@@ -364,67 +365,67 @@ export class CronJobService {
     await this.completeCronJob(cronJob);
   }
 
-  // @Cron('*/5 * * * *')
+  @Cron('*/2 * * * *')
   /**
    * Process an abuse webhook.
    * @returns {Promise<void>} - Returns a promise that resolves when the operation is complete.
    */
-  // public async processAbuse(): Promise<void> {
-  //   const isCronJobRunning = await this.isCronJobRunning(CronJobType.Abuse);
-
-  //   if (isCronJobRunning) {
-  //     return;
-  //   }
-
-  //   this.logger.log('Abuse START');
-  //   const cronJob = await this.startCronJob(CronJobType.Abuse);
-
-  //   try {
-  //     const webhookEntities = await this.webhookRepository.findByStatusAndType(
-  //       WebhookStatus.PENDING,
-  //       EventType.ABUSE_DETECTED,
-  //     );
-
-  //     for (const webhookEntity of webhookEntities) {
-  //       try {
-  //         const jobEntity =
-  //           await this.jobRepository.findOneByChainIdAndEscrowAddress(
-  //             webhookEntity.chainId,
-  //             webhookEntity.escrowAddress,
-  //           );
-  //         if (!jobEntity) {
-  //           this.logger.log(ErrorJob.NotFound, JobService.name);
-  //           throw new ControlledError(
-  //             ErrorJob.NotFound,
-  //             HttpStatus.BAD_REQUEST,
-  //           );
-  //         }
-  //         if (
-  //           jobEntity.escrowAddress &&
-  //           jobEntity.status !== JobStatus.CANCELED
-  //         ) {
-  //           await this.jobService.processEscrowCancellation(jobEntity);
-  //           jobEntity.status = JobStatus.CANCELED;
-  //           await this.jobRepository.updateOne(jobEntity);
-  //         }
-  //         await this.paymentService.createSlash(jobEntity);
-  //       } catch (err) {
-  //         this.logger.error(
-  //           `Error slashing escrow (address: ${webhookEntity.escrowAddress}, chainId: ${webhookEntity.chainId}: ${err.message}`,
-  //         );
-  //         await this.webhookService.handleWebhookError(webhookEntity);
-  //         continue;
-  //       }
-  //       webhookEntity.status = WebhookStatus.COMPLETED;
-  //       await this.webhookRepository.updateOne(webhookEntity);
-  //     }
-  //   } catch (e) {
-  //     this.logger.error(e);
-  //   }
-
-  //   this.logger.log('Abuse STOP');
-  //   await this.completeCronJob(cronJob);
-  // }
+  public async processAbuse(): Promise<void> {
```

Comment:
Most of the logic of this function should not be here, maybe makes sense to create a an abuse module.
Part of this code will also be called from content moderation module when a possitve abuse is detected.